test/services/notifications/setup/update-notifications.service.tests.js

```diff
@@ -0,0 +1,163 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const EventHelper = require('../../../support/helpers/event.helper.js')
+const { timestampForPostgres } = require('../../../../app/lib/general.lib.js')
+const ScheduledNotificationHelper = require('../../../support/helpers/scheduled-notification.helper.js')
+const ScheduledNotificationModel = require('../../../../app/models/scheduled-notification.model.js')
+
+// Thing under test
+const UpdateNotificationsService = require('../../../../app/services/notifications/setup/update-notifications.service.js')
+
+describe('Notifications Setup - Update Notifications service', () => {
+  let eventId
+  let notifications
+  let scheduledNotification
+  let scheduledNotification2
+
+  beforeEach(async () => {
+    const event = await EventHelper.add({
+      type: 'notification',
+      subtype: 'returnsInvitation'
+    })
+
+    eventId = event.id
+
+    scheduledNotification = await ScheduledNotificationHelper.add({
+      createdAt: timestampForPostgres(),
+      eventId,
+      status: 'sending'
+    })
+  })
+
+  describe('when updating a single notification', () => {
+    beforeEach(async () => {
+      notifications = [
+        {
+          createdAt: new Date(scheduledNotification.createdAt),
+          id: scheduledNotification.id,
+          notifyStatus: 'received',
+          status: 'sent'
+        }
+      ]
+    })
+
+    it("updates only the notification's required values", async () => {
+      await UpdateNotificationsService.go(notifications)
+
+      const createdResult = await ScheduledNotificationModel.query().where('eventId', eventId)
+
+      expect(createdResult).equal([
```

Comment:
Two things. We're updating not creating so `createdResult` doesn't read right in this context. To avoid the confusion altogether, just stick to our convention of `result` or `results` (depending on whether one or more results are being returned) in tests.

Secondly, I think you can make it even more verbose that your scheduled notification got updated by using a different syntax to get it.

```suggestion
      const result = await scheduledNotification.$query()

      expect(result).equal(
```